### PR TITLE
Feature/redux boiler

### LINKED ISSUE #2: Add redux Boilerplate
- [ ] add reducers
- [ ] add store

### PR DESCRIPTION
**Pull request by Ryan-D-Miller**
# Describe Changes

Please include changes made or which issues were fixed.

Add redux boiler plate with starting actions and reducers
closes #2 
# Type of Change
- [ ] Feature HTML
- [x] Feature JavaScript
- [ ] Feature CSS 
- [ ] Refactor/Fix Bugs

# Please describe the change made

Please give a short description of the changes that were made below
- Fix:

- Feature:
Add redux boiler plate

# How has this been tested?

- [x] In Browser
- [ ] In Console

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export const removeWish = (id) => {
```
Comment (daomeow) [inline src/actions/index.js:1]: Do we need to wrap this action in parentheses since we are returning an object?

File: src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    return {
```
Comment (daomeow) [inline src/actions/index.js:2]: If you want to remove this return we could wrap the whole action object in parentheses

File: src/actions/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        type: 'REMOVEWISH',
```
Comment (daomeow) [inline src/actions/index.js]: Do we want to format the type value with "loud snake case" (upcased and separated with underscores)?

File: src/index.js [hunk 4]
```diff
@@ -3,10 +3,18 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { createStore } from 'redux';
+import rootReducer from './reducers';
+import { Provider } from 'react-redux';
+
+const store = createStore(rootReducer,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
```
Comment (daomeow) [inline src/index.js:11]: Yay to Chrome extensions

Comment (daomeow) [inline src/index.js:11]: Yay to Chrome extensions

File: src/actions/index.js (hunk 2 above)
Comment (Ryan-D-Miller) [reply src/actions/index.js:2]: im weird and i like the return statement for some reason makes it easier to read for myself

File: src/reducers/wishListReducer.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+const wishListReducer = (state = [], action) => {
+    switch(action.type) {
```
Comment (errabun) [inline src/reducers/wishListReducer.js:2]: Like the use of switch and case instead of if/else!

**Review by errabun — COMMENTED** [review]
This looks great!  Awesome work team!

**Pull request merged**